src/platform_impl/linux/window.rs

```diff
@@ -185,10 +186,14 @@ impl Window {
 
     let settings = Settings::default();
 
+    let mut theme = Theme::Light;
```

Comment:
Instead of assuming that it is always light we should:
1. change the newly added `window.theme` to be `explicit_theme: Option<Theme>`
2. in the `Window.theme()` getter we check if `self.explicit_theme.unwrap_or_else(|| /* old logic here */)`

Reply:
OK, that also works in my use case. Since the configuration name is `preferred_theme`, I think it represents the field well. I'll update this branch.